demosplan/DemosPlanCoreBundle/Controller/Procedure/DemosPlanProcedureController.php

```diff
@@ -1345,6 +1345,10 @@ public function administrationEditAction(
                     $inData['r_name'] = $inData['r_externalName'];
                 }
 
+                // also change the procedureSettings EmailTitle as it contains the ProcedureName
+                // used for email subjects when inviting registered institutions
+                $inData['r_emailTitle'] = 'Einladung zur Beteiligung: '.$inData['r_name'];
```

Comment:
This should be a transkey

Reply:
We dont need this trans-key at all anymore.